setup/openstack/single-instance.yml

```diff
@@ -0,0 +1,91 @@
+heat_template_version: 2018-08-31
```

Comment:
Can you move the file to `dev-scripts` directory? `setup` is a new directory and it'll add clutter to the root of the project.

Reply:
Of course! Done.